setup.py

```diff
@@ -25,15 +22,8 @@
     # tornado will now only get installed and we'll get the newer one
     pass
 
-dev_requires = [
-    "flake8>=2.5",
-    "sphinx-rtd-theme>=0.1.9",
-    "sphinx>=1.3",
-    "pytest-reraise",
-    "pytest>=6.1.1,<8.2.0",
-] + tornado_requires
+dev_requires += tornado_requires
```

Comment:
I think this is not needed because `dev_requires` already containes `tornado_requires` in the toml file.